patches/extra/ungoogled-chromium/default-webrtc-ip-handling-policy.patch

```diff
@@ -0,0 +1,11 @@
+--- a/chrome/browser/ui/browser_ui_prefs.cc
++++ b/chrome/browser/ui/browser_ui_prefs.cc
+@@ -93,7 +93,7 @@ void RegisterBrowserUserPrefs(user_prefs
+                                 false);
+ #endif
+   registry->RegisterStringPref(prefs::kWebRTCIPHandlingPolicy,
+-                               blink::kWebRTCIPHandlingDefault);
```

Comment:
I think we should clearly document all changes to default behavior (maybe in a new document) along with how to change it back to Chromium's default behavior (if possible). That way us developers and users can have a baseline to compare against vanilla Chromium.

Reply:
There are occasional issues created by people running into some of the other changes to default settings, so this would be a good idea.  Though I think it'd be best to make that addition a separate PR considering how much is being done in this one already.